recipes/poco/all/patches/1.14.2.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/Foundation/CMakeLists.txt b/Foundation/CMakeLists.txt
+index 8508261e9..503db8b3d 100644
+--- a/Foundation/CMakeLists.txt
++++ b/Foundation/CMakeLists.txt
+@@ -99,7 +99,8 @@ set_target_properties(Foundation
+ )
+
+ if(POCO_UNBUNDLED)
+-	target_link_libraries(Foundation PRIVATE Pcre2::Pcre2 ZLIB::ZLIB Utf8Proc::Utf8Proc)
++	target_link_libraries(Foundation PRIVATE PCRE2::8BIT utf8proc::utf8proc)
++	target_link_libraries(Foundation INTERFACE ZLIB::ZLIB)
+ 	target_compile_definitions(Foundation PUBLIC POCO_UNBUNDLED)
+ else()
+ 	target_compile_definitions(Foundation PUBLIC UTF8PROC_STATIC)
+diff --git a/NetSSL_Win/CMakeLists.txt b/NetSSL_Win/CMakeLists.txt
+index c0e17689d..32a118719 100644
+--- a/NetSSL_Win/CMakeLists.txt
++++ b/NetSSL_Win/CMakeLists.txt
+@@ -21,7 +21,7 @@ set_target_properties(NetSSLWin
+ 	DEFINE_SYMBOL NetSSL_Win_EXPORTS
+ )
+
+-target_link_libraries(NetSSLWin PUBLIC Poco::Net Poco::Util Crypt32.lib)
++target_link_libraries(NetSSLWin PUBLIC Poco::Net Poco::Util crypt32 ws2_32)
```

Comment:
has this been reported upstream?  (if this is a pre-existing issue in the recipe, no need to address in this PR)